tests/php/test-amp-vimeo-embed.php

```diff
@@ -12,6 +12,15 @@
  */
 class AMP_Vimeo_Embed_Test extends WP_UnitTestCase {
 
+	/**
+	 * Tears down the environment after each test.
+	 *
+	 * @inheritDoc
+	 */
+	public function tearDown() {
+		remove_all_filters( 'do_shortcode_tag' );
```

Comment:
I believe this isn't necessary because filters get reset after each test anyway.

Reply:
OK, sounds good.

Reply:
https://github.com/ampproject/amp-wp/pull/3678/commits/3e0e649706533cbe6fbf96cb5c63710f32887d93#diff-58db4548e89d6f4290514ec825c906ecL15 removes this `tearDown()` method.